bots/image-refresh

```diff
@@ -133,17 +150,35 @@ def run(image, verbose=False, **kwargs):
 
     branch = task.branch(image, "images: Update {0} image".format(image), pathspec="bots/images", **kwargs)
     if branch:
-        pull = task.pull(branch, **kwargs)
 
-        # we want to validate all image refreshes with external projects
-        task.label(pull, ["test-external"])
+        pull = task.pull(branch, run_tests=False, **kwargs)
 
         # Trigger this pull request
         api = github.GitHub()
+        old_head = pull["head"]["sha"]
         head = api.get("pulls/{}".format(pull["number"]))["head"]["sha"]
+        retry = 3
```

Comment:
It seems like this should be moved into `task.pull()`, so that every created PR waits until the API is consistent.